scanomatic/ui_server/data_api.py

```diff
@@ -17,16 +16,16 @@
 from scanomatic.data_processing.norm import NormState
 from scanomatic.data_processing.project import path_has_saved_project_state
 from scanomatic.image_analysis.first_pass_image import FixtureImage
-from scanomatic.image_analysis.grayscale import (
+from scanomatic.image_analysis.grayscale import \
```

Comment:
Oh? Interesting.

Reply:
That's the automatic formatting script does that unfortunately but will fix